test/testlib/minikube_utilities.go

```diff
@@ -459,3 +476,34 @@ func RunSQL(t *testing.T, namespace string, podName string, databaseName string,
 
 	return result, err
 }
+
+func executeCommandsInPod(t *testing.T, podName string, namespaceName string, commands []string) {
```

Comment:
should this function be exported from the package? (capital vs lowercase e)

Reply:
Done